pontoon/base/static/js/translate.js

```diff
@@ -970,12 +970,16 @@ var Pontoon = (function (my) {
         return;
       }
 
+      if ($('#from').val() || $('#to').val()){
+        return;
+      }
+
```

Comment:
We should still update the chart, so we cannot return.

Reply:
the #rangepicker chart ?